lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -1234,7 +1234,7 @@ class ConvertAtenMatmulBaseOp : public OpConversionPattern<AtenOpT> {
       return false;
     };
 
-    SmallVector<TensorShape_t> commonElems, lhsSqueezedElems, rhsSqueezedElems;
```

Comment:
why was this named commonElems originally?

Reply:
The name was pretty unclear - I think what they meant was dimensions commonly shared between the LHS and RHS but not contracted on. By definition these are the batch dimensions so I just renamed.